app/code/Magento/CustomerImportExport/Test/Unit/Model/Export/AddressTest.php

```diff
@@ -181,16 +205,16 @@ protected function _getModelDependencies()
      */
     public function getWebsites($withDefault = false)
     {
-        $websites = [];
+        $websites = $this->_websites;
```

Comment:
Looks like not correct change, but originally it wasn’t removing anything either. I think it’s better to to change it

Reply:
Yeah, whole block 
```php
        if (!$withDefault) {
            unset($websites[0]);
        }
```

should be just removed, the assignment should be still `$websites = [];`

Reply:
As we discussed in Slack - please just revert these changes in order to cut the scope of the PR. Let's fix it separately

Reply:
Then the tests are going to fail. As it's trying to remove non-existing item.
As these methods are not being used anywhere, I'll just remove them and I hope that @magento/community-engineering-team  won't try to keep dead code.